{{ cookiecutter.project_slug }}/README.uv.jinja

```diff
@@ -0,0 +1,87 @@
+This is a Python application that uses [uv] for packaging and dependency management.
+It  provides [`pre-commit`][pre-commit] hooks for various linters
+and formatters and automated tests using [`pytest`][pytest] and [GitHub Actions].
+Pre-commit hooks are automatically kept updated with a dedicated GitHub Action, this can
+be removed and replaced with [pre-commit.ci] if using a public repo.
+
+It was developed by the [Imperial College Research Software Engineering Team].
+
+## Usage
+
+To get started:
+
+1. [Download and install uv] following the instructions for your OS.
+2. Set up the virtual environment:
+
+   ```bash
+   uv venv .venv
```

Comment:
I think it's simpler to just use the `uv sync` command straight away to both create the virtual environment _and_ install the packages straight away.
```suggestion
   uv sync
```